icu4c/source/test/cintltst/cloctst.c

```diff
@@ -6202,6 +6202,11 @@ static void TestLikelySubtags(void)
         const char* const minimal = full_data[i][0];
         const char* const maximal = full_data[i][1];
 
+		if (uprv_strcmp(minimal, "und_Hant_CN") == 0 &&
```

Comment:
Please go through the source code changes and replace tabs with spaces.

Reply:
I think I caught them all. Sorry!